Common/src/main/resources/assets/hexcasting/lang/en_us.flatten.json5

```diff
@@ -901,6 +901,7 @@
       invalid_spell_datum_type: "Tried to use a value of invalid type as a SpellDatum: %s (class %s). This is a bug in the mod.",
       unknown: "threw an exception (%s). This is a bug in the mod.",
       shame: "Shame on you!",
+      stack_size: "My mind was too small to contain the breadth of this spell.",
```

Comment:
I'd argue this should be a bit less flowery, and actually say what the player did wrong (ie. exceed the stack size limit), to match other error messages.

Reply:
made more literal